packages/plugin-apex/test/commands/force/apex/execute.test.ts

```diff
@@ -90,6 +92,24 @@ describe('force:apex:execute', () => {
   const compileResponse =
     'Error: Line: 11, Column: 1\nError: problem compiling\n\n';
   const runtimeResponse = `Compiled successfully.\nError: problem at runtime\nError: Issue in mock file\n\n${log}\n`;
+  const TEST_USERNAME = 'test@org.com';
+
+  let sandboxStub: SinonSandbox;
+
```

Comment:
figured this out through debugging into the core/command code and looking at existing parking orbit PRs. Like
 - https://github.com/salesforcecli/plugin-data/pull/284
 - https://github.com/salesforcecli/plugin-user/pull/319/files